snmp_cisco/manifest.json

```
@@ -0,0 +1,33 @@
+{
+  "display_name": "Cisco",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "snmp_cisco",
+  "metric_prefix": "snmp.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Collect SNMP metrics from your CISCO network devices.",
+  "guid": "4109f288-7460-4ed2-b0c3-04e708fbd5bd",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Cisco Integration",
+  "categories": [
+    ""
```

```suggestion
    ""
```

This seems not correct. I think you can maybe use the same ones used here: https://github.com/DataDog/integrations-core/blob/7adb08342bbfd0eff59fbff64c11fbaf4b766c85/snmp/manifest.json#L2-L7

This is used for filtering in Doc UI: https://docs.datadoghq.com/integrations/

cc @naaltman @DataDog/documentation

---

👍 maybe also include snmp here?